src/ci/run.sh

```diff
@@ -102,6 +102,18 @@ else
   fi
 
   RUST_CONFIGURE_ARGS="$RUST_CONFIGURE_ARGS --set rust.verify-llvm-ir"
+
+  # We enable this for non-dist builders, since those aren't trying to produce
+  # fresh binaries. We currently don't entirely support distributing a fresh
+  # copy of the compiler (including llvm tools, etc.) if we haven't actually
+  # built LLVM, since not everything necessary is copied into the
+  # local-usage-only LLVM artifacts. If that changes, this could maybe be made
+  # true for all builds. In practice it's probably a good idea to keep building
+  # LLVM continuously on at least some builders to ensure it works, though.
+  # (And PGO is it's own can of worms).
```

Comment:
Agreed this doesn't need to happen in this PR. I definitely think we should (at some point) make sure the pre-compiled LLVM has all the binaries / libraries you'd get if you built locally though, people have been running into various issues related to that.
```suggestion
  # (And PGO is its own can of worms).
```